lightning/src/events/mod.rs

```diff
@@ -1176,7 +1192,7 @@ pub enum Event {
 		/// The channel value of the requested channel.
 		funding_satoshis: u64,
 		/// Our starting balance in the channel if the request is accepted, in milli-satoshi.
```

Comment:
This field is explained above, but this field-comment is a bit obsolete now.

Reply:
Yeah good find. It's the old `push_msat` doc. I will update it.